libzhlgen/test/functions/mac/1.6.13/Ship.zhl

```diff
@@ -0,0 +1,73 @@
+"NOT LISTED IN GHIDRA":
+cleanup __amd64 Globals::Ellipse Ship::GetBaseEllipse(Ship *this); /* This comment here wasn't made by Dino ---> /* TODO: Ellipse is actually returned on RAX:XMM0 or maybe it's RDX:XMM0 */
```

Comment:
You can remove the comment, it's a leftover from when I was doing Linux64 and at first I used to have to manually specify the return registers until I changed it around to just letting the compiler handle it and not having to specify these